_docker/app/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM php:8.3.6-fpm
+
+RUN apt-get update && apt-get install -y \
+      apt-utils \
+      libpq-dev \
+      libpng-dev \
+      libzip-dev \
+      zip unzip \
+      git && \
+      docker-php-ext-install pdo_mysql && \
```

Comment:
As a recommendation, separate the blocks when installing php modules from apt and from docker-php-ext